FV3/atmos_cubed_sphere/model/fv_arrays.F90

```diff
@@ -1025,6 +1025,8 @@ module fv_arrays_mod
    logical :: write_coarse_restart_files = .false.  ! Whether to write coarse restart files
    logical :: write_coarse_diagnostics = .false.  ! Whether to enable writing coarse diagnostics
    logical :: write_only_coarse_intermediate_restarts = .false.  ! Whether to write only coarse intermediate restart files (if do_coarse_graining is .true.)
+   logical :: restart_from_agrid_winds = .false.  ! Whether to restart from A-grid winds
+   logical :: optional_dgrid_vel_rst = .false.  ! Whether to write out optional D-grid winds when restart_from_agrid_winds is active
```

Comment:
This option might be confusing for someone not familiar with the goals of this, but with good comments like this should be ok? It seems clear to me, but I already have some context :)

Reply:
Hopefully it's a little clearer based on Oli's suggestion; I went with the more verbose option, `write_optional_dgrid_vel_rst`.  I agree that it is probably a little off-putting to someone without more context.